css/cssom/CSSStyleSheet-constructable-concat.html

```diff
@@ -4,14 +4,20 @@
 <link rel="author" href="mailto:masonf@chromium.org">
 <link rel="help" href="https://drafts.csswg.org/cssom/#extensions-to-the-document-or-shadow-root-interface">
 <link rel="match" href="CSSStyleSheet-constructable-concat-ref.html">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
 
 <span>This should be green</span>
 <div id=host></div>
+<pre id=errors></pre>
 
 <script>
-test(function() {
+function assert_equals(val1,val2) {
+  if (val1 === val2) {
+    return;
```

Comment:
I'm concerned that if something in the JS throws before `assert_equals()`, the checks will not run and it may be mistaken for a pass.

I think it would be better for `assert_equals()` to also write something in case of a pass, or maybe use `window.onerror` to detect errors.